pkgs/tools/networking/nss-mdns/default.nix

```
@@ -1,14 +1,17 @@
-{ fetchurl, lib, stdenv, fetchpatch }:
+{ fetchurl, lib, autoreconfHook, pkg-config, stdenv, fetchpatch, fetchFromGitHub }:
 
 stdenv.mkDerivation rec {
   pname = "nss-mdns";
-  version = "0.10";
+  version = "v0.15.1";
 
-  src = fetchurl {
-    url = "http://0pointer.de/lennart/projects/nss-mdns/nss-mdns-${version}.tar.gz";
-    sha256 = "0vgs6j0qsl0mwzh5a0m0bykr7x6bx79vnbyn0r3q289rghp3qs0y";
+  src = fetchFromGitHub {
+    owner = "lathiat";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-iRaf9/gu9VkGi1VbGpxvC5q+0M8ivezCz/oAKEg5V1M=";
```

```suggestion
    hash = "sha256-iRaf9/gu9VkGi1VbGpxvC5q+0M8ivezCz/oAKEg5V1M=";
```

Nixpkgs is slowly moving to SRI hashes, thus the `hash` attribute now.